### PR TITLE
[WIP] use spectron for test

### PR DESCRIPTION
**Pull request by AlanJager**
[--->more information](https://github.com/electron/spectron/blob/master/README.md)

### CODE REVIEW

File: test/testWindowNum.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+var Application = require('spectron').Application
```
Comment (KagamiChan) [inline test/testWindowNum.js]: could be `const` here, avoiding `var` is recommended

**Comment by AlanJager** [comment]
@KagamiChan I will write more test cases for reviewing

**Comment by KagamiChan** [comment]
@AlanJager OK, added a WIP tag for PR title.
Are you using eslint plugin/integration for youe code editor? Pls consider enable it

**Comment by KagamiChan** [comment]
ping @AlanJager 
Is there any update since my last comment?

**Comment by AlanJager** [comment]
@KagamiChan I'm still working on a local repo aims at the integration test for poi. And will update the commit when it works really well.

**Comment by KagamiChan** [comment]
@AlanJager Glad to hear that, thank you for you work. You may leave this issue open anyway, or create pull requests when you feel like to. We can do it step by step, file after file, since the evolution on master branch may make if hard to rebase.

**Pull request closed without merging**